### DISCUSSION TITLE
Gap between title bar and content area

### DISCUSSION THREAD

**Discussion by theFG1998** [kovidgoyal/kitty#5812] [Q&A] [answered]
<img width="1508" alt="Screenshot 2022-12-23 at 10 59 33" src="https://user-images.githubusercontent.com/66351806/209264679-e0ce928b-d872-43d8-8e6d-9ab86326b3e2.png">

Any way to solve this?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.placement_strategy

**Reply by theFG1998** [reply]
> https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.placement_strategy

thank you!
